src/ParkingAPI.java

```diff
@@ -2,14 +2,30 @@
 
 public class ParkingAPI {
 
-    
+	public ParkingAPI() {
 
-    public ParkingAPI() {
+	}
 
-    }
+	public int getOpenSpots(ParkingLot parkingLot) {
+		Random random = new Random(parkingLot.ordinal());
+		return random.nextInt(110);
+	}
+	
+	public int getFilledSpots(ParkingLot parkingLot) {
+		Random random = new Random(parkingLot.ordinal());
+		return random.nextInt(110);
+	}
 
-    public int getOpenSpots(ParkingLot parkingLot) {
-        Random random = new Random(parkingLot.ordinal());
-        return random.nextInt(110);
-    }
-}
\ No newline at end of file
+	public ParkingLot availableParkingLots(ParkingLot one, ParkingLot two) {
+		int availableLotOne = getOpenSpots(one) - getFilledSpots(one);
+		int availableLotTwo = getOpenSpots(two) - getFilledSpots(two);
```

Comment:
Can we change some of this logic?

With this logic, say a parking lot has 10 spots. If 4 are filled, then `getOpenSpots()` would return `6`, and `getFilledSpots()` would return `4`, and then this code would claim that there are only 2 open spots in the lot.